index.js

```diff
@@ -11,21 +10,26 @@ class KEAFacility extends BaseFacility {
     this._hasConf = true
     this.leases = []
     super.init()
-    this.fetchServerConf()
   }
 
-  sendCommand (command, service, args = undefined) {
+  setNetFac (netFac) {
+    this.netFac = netFac
+    this.fetchConf()
```

Comment:
hmm not sure why these two are connected, shouldn't setNetFac simply set the http client?

Reply:
I think you misunderstood, so setNetFac is ok but this.fetchConf didn't make much sense there, now this.netFac is not set anywhere

Reply:
Hey, ya removed the function and doing `this.kea_k0.netFac = this.http_c0` on the worker. Didn't see the need for the function after splitting the logic